backend/de.metas.invoice.webui/src/main/java/de/metas/invoicecandidate/process/C_Invoice_Candidate_EnqueueSelectionForInvoicing.java

```diff
@@ -188,6 +206,32 @@ public LookupValuesList getAvailableForexContracts()
 				: LookupValuesList.EMPTY;
 	}
 
+	@ProcessParamLookupValuesProvider(parameterName = InvoicingParams.PARA_BP_BankAccountId, numericKey = true, lookupSource = DocumentLayoutElementFieldDescriptor.LookupSource.lookup)
+	public LookupValuesList getAvailableBankAccountIds(@NonNull final LookupDataSourceContext context)
```

Comment:
won't this thrown an exception if `getInvoicingParams().isInvoicingSingleBPartner()==false` and the user tries to invoice multiple partners?

Reply:
nope, only if the user tries to fill in a Bank Account, which is not supported for multiple BPs

Reply:
also, the Bank Account parameter is not displayed in standard metasfresh